### PR TITLE
Migrate ContentButton to MAUI base on NUI

### PR DESCRIPTION
**Pull request by shyunMin**
### Description of Change ###
This PR is to prepare migrating TV Controls to MAUI(NUI). It includes below changes
 - Update package from `Tizen.UIExtensions.Elmsharp` to `Tizen.UIExtensions.NUI`.
 - Update support plaftorm version from 6.5 to 7.0.
 - Comment out controls to prevent build errors.
 - Check build test and run `Sample` app.
 - Migrate `ContentButton` control to MAUI(NUI) and test -> https://github.com/Samsung/Tizen.TV.UIControls/pull/95/commits/6e842e2a303cd721ebbf60c177d134a73d49f136

### Status
|Test|status|
|-|-| 
|Shadow Frame|  |
|Page Transition|  |
|MediaPlayer|  |
|RemoteControl|  |
|DrawerLayout|  |
|Editor|  |
|Entry|  |
|CustomFocus|  |
|ContentButton| done |
|ContentPopup| |
|Grid Test| |

### CODE REVIEW

**Comment by JoonghyunCho** [comment]
Is it ok to remove all the resources?

**Comment by JoonghyunCho** [comment]
What are all the commented codes?

File: src/Tizen.Theme.Common/Renderer/ContentButtonHandler.cs [hunk 1]
```diff
@@ -17,56 +17,45 @@
...
 using Microsoft.Maui.Handlers;
 using Microsoft.Maui.Platform;
-using Tizen.UIExtensions.Common;
-using Tizen.UIExtensions.ElmSharp;
-using TButton = Tizen.UIExtensions.ElmSharp.Button;
+using Tizen.NUI;
+using Tizen.UIExtensions.NUI;
+using NTouchEventArgs = Tizen.NUI.BaseComponents.View.TouchEventArgs;
 
 namespace Tizen.Theme.Common.Renderer
```
Comment (sung-su) [inline src/Tizen.Theme.Common/Renderer/ContentButtonHandler.cs]: Need to change namespace too.

**Comment by shyunMin** [comment]
> Is it ok to remove all the resources?

Yes, the resources have already been copied to the `Sample` project in #93 for migrating MAUI before.
I removed the duplicated files that hadn't been deleted yet.

**Comment by shyunMin** [comment]
> What are all the commented codes?

The classes that commented out are that need to be implemented.
For build and sample app test,  I have just commented out.

File: sample/Sample.Tizen/Sample.Tizen.csproj [hunk 2]
```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Tizen.NET.Sdk/1.0.9">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>tizen40</TargetFramework>
+    <TargetFramework>net6.0-tizen</TargetFramework>
   </PropertyGroup>
```
Comment (rookiejava) [inline sample/Sample.Tizen/Sample.Tizen.csproj]: no need to specifty the MAUI Version here ? (for a consistent develpment environment)

**Review by rookiejava — COMMENTED** [review]
It would be great if you could change the Sample project to MAUI's single project style not XF Franken proj style.(Core and Platform project)

**Comment by JoonghyunCho** [comment]
> 

Thanks for your answer...

File: sample/Sample/Sample.csproj [hunk 3]
```diff
@@ -6,11 +6,11 @@
     <UseMaui>true</UseMaui>
     <SingleProject>true</SingleProject>
     <ImplicitUsings>enable</ImplicitUsings>
-    <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'tizen'">6.5</SupportedOSPlatformVersion>
+    <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'tizen'">7.0</SupportedOSPlatformVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Tizen.UIExtensions.ElmSharp" Version="0.8.1" />
+    <PackageReference Include="Tizen.UIExtensions.NUI" Version="0.9.0" />
```
Comment (rookiejava) [inline sample/Sample/Sample.csproj]: In MAUI app, no direct UIExtension dependency requried anymore.

File: src/Tizen.Theme.Common/Renderer/ContentButtonHandler.cs (hunk 1 above)
Comment (shyunMin) [reply src/Tizen.Theme.Common/Renderer/ContentButtonHandler.cs]: Fixed :)

File: sample/Sample.Tizen/Sample.Tizen.csproj (hunk 2 above)
Comment (shyunMin) [reply sample/Sample.Tizen/Sample.Tizen.csproj]: Thanks for your suggestion. I've updated project style and removed unnecessary files.

File: sample/Sample/Sample.csproj (hunk 3 above)
Comment (shyunMin) [reply sample/Sample/Sample.csproj]: I've removed :)

**Review by sung-su — APPROVED** [review]
lgtm

**Review by JoonghyunCho — APPROVED** [review]

**Pull request merged**